README.md

```diff
@@ -323,31 +124,47 @@ $ tendermint lite
 
 ## 8. Send Your First Transaction
 
-Genesis funds are bonded funds, to transfer freely around, you first have to withdraw to UTXO
-
-- Create Transfer Address
+ Before doing any transaction, It is noted that, the genesis fund is **bonded** at the beginning. To move funds freely around, we first have to withdraw it to UTXO:
```

Comment:
probably a bit more precise is that it's staked (some of the initial genesis amounts are bonded, some of them are unbonded)

Reply:
Fixed, thanks Tomas